massa-execution/src/vm.rs

```diff
@@ -139,6 +139,12 @@ impl VM {
     ) -> SCELedgerChanges {
         let mut context = self.execution_context.lock().unwrap();
 
+        // credit the sender with "coins"
```

Comment:
It would be great if we could refer/link to a spec.

Reply:
done